### PR TITLE
[Docker-compose] Deployment - Include Device-basic-auth and  http-agent-redis

### PR DESCRIPTION
**Pull request by danielapetito**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
Deployment - Include Device-basic-auth and  http-agent-redis

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:
dojot/dojot#2370

### CODE REVIEW

**Review by ewerton-silva00 — DISMISSED** [review]
Ok.

File: docker-compose.yml [hunk 1]
```diff
@@ -1206,6 +1216,28 @@ services:
   #     - 5050:80
 # GUI to postgres - END
 
+  device-basic-auth:
```
[the remark this answers is not in the document]
Comment (danielapetito) [reply docker-compose.yml:1218]: ok.. updated

File: docker-compose.yml [hunk 2]
```diff
@@ -1206,6 +1216,28 @@ services:
...
+    restart: always
+    depends_on:
+      - kafka
+      - data-broker
+      - mongodb
+      - auth
+      - device-manager
+    ports:
+      - 3000:3000
+      - 9229:9229
```
[the remark this answers is not in the document]
Comment (danielapetito) [reply docker-compose.yml:1229]: ok.. updated

File: docker-compose.yml [hunk 3]
```diff
@@ -918,6 +918,7 @@ services:
 
   http-agent:
     image: dojot/http-agent:${DOJOT_VERSION}
+    # image: dojot/http-agent:v0.7.1-rc.1
     depends_on:
       - kafka
```
[the remark this answers is not in the document]
Comment (danielapetito) [reply docker-compose.yml]: ok.. updated

**Review by ewerton-silva00 — APPROVED** [review]
Ok.

**Review by gilmaicor — APPROVED** [review]
Good job!

**Pull request merged**